data/abilities.ts

```diff
@@ -182,7 +182,15 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 			this.add('-ability', pokemon, 'As One');
 			this.add('-ability', pokemon, 'Unnerve', pokemon.side.foe);
 		},
-		onFoeTryEatItem: false,
+		onEnd() {
+			this.effectData.unnerved = false;
+		},
+		onFoeTryEatItem() {
+			if (!this.effectData.unnerved) {
```

Comment:
These can be simplified to something like ``return !this.effectData.unnerved``, right?

Reply:
yes.